src/main/resources/fine_tuning/collections.yaml

```diff
@@ -13,7 +13,7 @@ collections:
         - -storeDocvectors
         - -storeRaw
       collection: TrecCollection
-      generator: JsoupGenerator
+      generator: DefaultLuceneDocumentGenerator
```

Comment:
Actually I was a bit hesitant about the changes here as well. As you mentioned here https://github.com/castorini/anserini/pull/1355#issuecomment-675485158 @lintool, updating non-running code would be a nightmare to verify. In this case, do we want to keep `fine_tuning`'s doc updated?